proto/vm/vm.proto

```diff
@@ -116,6 +116,8 @@ message InitializeRequest {
   // the messenger, keystore, shared memory, blockchain alias,
   // subnet alias, and appSender services
   string server_addr = 14;
+  // network_upgrades_bytes is the json encoded network upgrades
+  bytes network_upgrades_bytes = 15;
```

Comment:
It seems like an anti-pattern to encode a json blob into the proto. Can we create an upgrade message that specifies the actual upgrade times?

Reply:
added